pkg/logql/metrics.go

```diff
@@ -98,6 +100,7 @@ func RecordMetrics(ctx context.Context, p Params, status string, stats stats.Res
 		"step", p.Step(),
 		"duration", time.Duration(int64(stats.Summary.ExecTime*float64(time.Second))),
 		"status", status,
+		"query_tags", queryTags,
```

Comment:
I would actually try to output those as:

`status=success source=Value limit=1000`

instead of

`status=success query_tags=\"Source=logvolhist\" limit=1000`

Reply:
Ah. good point! @cyriltovena . I think I understand your concern!. But few questions.
1. You mean we don't need any differentiation that its `query_tags`? (e.g: If we just put `source=Value`, we may not know where this source comes from? From what header.
2. How do you want to log multiple values for `X-Query-Tags` header? (say `X-Query-Tags: Source=Foo,Feature=Beta`)
`status=success source=foo feature=beta`?

Reply:
@cyriltovena Fixed it!. It now logs in the desired format `status=success source=Value limit=1000`